packages/block-library/src/separator/save.js

```diff
@@ -1,3 +1,36 @@
-export default function save() {
-	return <hr />;
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	getColorClassName,
+} from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const {
+		backgroundColor,
+		customBackgroundColor,
+	} = attributes;
+
+	const backgroundClass = getColorClassName( 'background-color', backgroundColor );
+	const textClass = getColorClassName( 'color', backgroundColor );
+
+	const separatorClasses = classnames( {
+		'has-text-color has-background': backgroundColor || customBackgroundColor,
+		[ backgroundClass ]: backgroundClass,
+		[ textClass ]: textClass,
```

Comment:
Can you maybe add a comment explaining why you're using the same color for both text and background colors?